### ISSUE TITLE
How can I get outputs with labels in Classifier Free Guidance?

### ISSUE THREAD

**Issue by HyperGDX** [lucidrains/denoising-diffusion-pytorch#123]
Hi,

I want to use the Classifier Free Guidance diffusion model to generate some images with labels.
I have used my own dataset to train the Classifier Free Guidance diffusion model, but when I use func sample in class GaussianDiffusion, it only return pred_x without its label. How can I solve this problem?

``` python
def generate(self, milestone, img_classes):
    self.load(milestone)
    with torch.no_grad():
        sampled_images = self.model.sample(
            classes=image_classes,
            cond_scale=3.                # condition scaling, anything greater than 1 strengthens the classifier free guidance. reportedly 3-8 is good empirically
        )
    return sampled_images

```
``` python
gen = trainer.generate(milestone=17, img_classes=torch.randint(0, 176, (8,)).cuda())
```

Thanks and regards.

**Comment by lucidrains** [comment]
@HyperGDX you have the labels, since you passed them in as conditioning as `img_classes`

**Comment by HyperGDX** [comment]
Thanks.
